services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/security/AccessRules.kt

```diff
@@ -0,0 +1,17 @@
+package fi.hel.haitaton.hanke.security
+
+import org.springframework.http.HttpMethod
+import org.springframework.security.config.annotation.web.builders.HttpSecurity
+
+class AccessRules {
+    companion object {
+        fun configureHttpAccessRules(http: HttpSecurity) {
+            http.anonymous().and()
+                    .authorizeRequests()
+                    .mvcMatchers(HttpMethod.GET, "/organisaatiot").permitAll()
+                    .mvcMatchers(HttpMethod.POST, "/hankkeet", "/hankkeet/**").hasRole("haitaton-user")
```

Comment:
Why do we have different access control for organisaatiot-api?

Reply:
No idea, but I do remember that we did talk about these restrictions and it was mentioned that organisaatiot-list access does not need the same tighter restrictions as the other stuff. Also, this list was copied from the main configuration, so this isn't new.. just in a new place.